pipelines/collate_plots.py

```diff
@@ -167,10 +190,17 @@ def collate_from_all_subdirs(
         One or more PDFs filenames to look for in the
         subdirectories and merge.
 
+    save_dir
+        Directory in which to save the merged PDFs.
+        If ``None``, use a "figures" directory in the parent directory of ``model_base_dir``. Default ``None``.
+
     Returns
     -------
     None
     """
+    if save_dir is None:
+        save_dir = Path(model_base_dir).parent / "figures"
+
```

Comment:
As above, create if not exists?

Reply:
This function eventually calls `merge_pdfs_from_subdirs`, which makes the directory.